quantum/audio/song_list.h

```diff
@@ -161,15 +161,9 @@
 #ifndef IN_LIKE_FLINT
 #    define IN_LIKE_FLINT
 #endif
-#ifndef IMPERIAL_MARCH
-#    define IMPERIAL_MARCH
-#endif
-#ifndef BASKET_CASE
-#    define BASKET_CASE
-#endif
-#ifndef COIN_SOUND
-#    define COIN_SOUND
-#endif
+#define IMPERIAL_MARCH HD_NOTE(_A4), HD_NOTE(_A4), HD_NOTE(_A4), QD_NOTE(_F4), QD_NOTE(_C5), HD_NOTE(_A4), QD_NOTE(_F4),  QD_NOTE(_C5), WD_NOTE(_A4), HD_NOTE(_E5), HD_NOTE(_E5), HD_NOTE(_E5), QD_NOTE(_F5), QD_NOTE(_C5), HD_NOTE(_A4), QD_NOTE(_F4),  QD_NOTE(_C5), WD_NOTE(_A4)
```

Comment:
Revert all of these.   They have been removed because they are copyrighted music, and to avoid any possibility of issues. 

You can create a `user_song_list.h` file and include them in that, if you need.